cpu/stm32_common/doc.txt

```diff
@@ -0,0 +1,55 @@
+/**
+ * @ingroup cpu
+ * @defgroup cpu_stm32_common STM32 common
+ * @{
```

Comment:
yep, I think the block is not needed, too - though it doesn't hurt either. Still I would re-order that to:

```
/**
 * @defgroup cpu_stm32_common STM32 common
 * @ingroup cpu
 * @brief STM32 common configurations
[ * @{ ] <- optional
 *
```

the `@brief` should be above `@{` (if still used)